api/src/main/java/org/openmrs/module/sync2/Sync2ModuleActivator.java

```diff
@@ -19,12 +21,13 @@
 public class Sync2ModuleActivator extends BaseModuleActivator {
 	
 	private static final Logger LOGGER = LoggerFactory.getLogger(Sync2ModuleActivator.class);
-	
+
 	/**
 	 * @see #started()
 	 */
 	public void started() {
 		LOGGER.info("Started Sync2 Module");
+		Context.getRegisteredComponents(SyncSchedulerServiceImpl.class).get(0).runSyncScheduler();
```

Comment:
This will be a nitpick. To be super correct 
```
LOGGER.info("Started Sync2 Module");
``` should be on the end of started method I think. So it displays the message only if no exceptions were thrown in the started method